src/atomate2/abinit/schemas/calculation.py

```diff
@@ -237,12 +291,40 @@ def from_abinit_files(
         abinit_gsr_file = dir_name / abinit_gsr_file
         abinit_log_file = dir_name / abinit_log_file
         abinit_abort_file = dir_name / abinit_abort_file
+        abinit_out_file = dir_name / abinit_out_file
+        abinit_outddb_file = dir_name / abinit_outddb_file
+        abinit_outpot_file = dir_name / abinit_outpot_file
+
+        abinit_objects: dict[AbinitObject, Any] = {}
+        if abinit_outddb_file.exists():
```

Comment:
I added a variable ```files_to_store``` which is a list of the extensions of the files to be saved if needed.

Reply:
I can't find the `files_to_store` variable. Where is it? Should it still be pushed?

Reply:
sorry, not sure why I did not find it 😄

Reply:
this might be because the file you were looking at in this PR was outdated? At some point I did not find it either depending from where I was opening the file